packages/neutron-sdk/src/interchain_queries/helpers.rs

```diff
@@ -98,3 +121,72 @@ pub fn create_validator_key<AddrBytes: AsRef<[u8]>>(
 
     Ok(key)
 }
+
+/// Creates Cosmos-SDK distribution key for fee pool
+/// https://github.com/cosmos/cosmos-sdk/blob/35ae2c4c72d4aeb33447d5a7af23ca47f786606e/x/distribution/types/keys.go#L46
+pub fn create_fee_pool_key() -> NeutronResult<Vec<u8>> {
+    let key: Vec<u8> = vec![FEE_POOL_KEY];
+
+    Ok(key)
+}
+
+/// Creates Cosmos-SDK governance key for proposal with specific id
+/// https://github.com/cosmos/cosmos-sdk/blob/35ae2c4c72d4aeb33447d5a7af23ca47f786606e/x/gov/types/keys.go#L41
+pub fn create_gov_proposal_key(proposal_id: u64) -> NeutronResult<Vec<u8>> {
+    let mut key: Vec<u8> = vec![PROPOSALS_KEY_PREFIX];
+    key.extend_from_slice(proposal_id.to_be_bytes().as_slice());
+
+    Ok(key)
+}
+
+/// Returns validator max change rate
+pub fn get_max_change_rate(commission: &Option<ValidatorCommission>) -> Option<Decimal> {
+    let commission_rates = commission.as_ref().map(|v| v.commission_rates.as_ref())?;
+    commission_rates.map(|v| Decimal::from_str(v.max_change_rate.as_str()).unwrap_or_default())
+}
```

Comment:
In this function (and also in several following functions), we return `Decimal::default()` even when incoming data (`v.max_change_rate.as_str()`) is invalid. Is it ok?

Reply:
Good question. I think it is ok, because `max_change_rate` should not be empty string and should contain number. It can be 0, but you can not create validator without `max_change_rate` as it is required parameter. I do not think that it is worth to throw exception because this situation is possible in the custom `staking` module only.

Reply:
alright, so here we trust staking module to work well. if it doesn't work well — we are doomed anyway.